packages/rum/test/longTaskCollection.ts

```diff
@@ -0,0 +1,44 @@
+import { LifeCycleEventType } from '../src/lifeCycle'
```

Comment:
Other files are suffixed by `*.spec.ts`, should we stick to this convention and name this one `longTaskCollection.spec.ts`?